base_utils.py

```diff
@@ -263,5 +263,6 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
             )
             all_dict_meta+= result['post_meta']
         except Exception as e:
```

Comment:
Medium severity: Broad exception catch risks masking errors. Catch specific exceptions:
```suggestion
except (KeyError, ValueError) as e:
    logger.error(f"Batch processing failed: {batch}, error: {e}")
```